akka-http/src/main/java/akka/http/javadsl/coding/Coder.java

```diff
@@ -25,9 +29,40 @@
         this.underlying = underlying;
     }
 
+    public HttpResponse encodeMessage(HttpResponse message) {
```

Comment:
Hmm, might just not work. But let's not reimplement any stuff here. Why not just `(HttpResponse) underlying.encodeMessage(message)`?